Subject: [PATCH] fix panther-x2 uboot
---
Index: arch/arm/dts/rk3566-panther-x2-u-boot.dtsi
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/arch/arm/dts/rk3566-panther-x2-u-boot.dtsi b/arch/arm/dts/rk3566-panther-x2-u-boot.dtsi
--- a/arch/arm/dts/rk3566-panther-x2-u-boot.dtsi	(revision 8ef46eca64286a30e5a2c071db316b7b5ae42a68)
+++ b/arch/arm/dts/rk3566-panther-x2-u-boot.dtsi	(revision 4f527e346d731d035bed6b3571f4b05a28c89327)
@@ -9,7 +9,7 @@
 / {
 	chosen {
 		stdout-path = &uart2;
-		u-boot,spl-boot-order = &sdmmc0, &sdhci, &spiflash;
+		u-boot,spl-boot-order = &sdhci, &sdmmc0, &sdmmc1;
 	};
 };
 
@@ -28,10 +28,6 @@
 &emmc_datastrobe {
 	bootph-all;
 };
-
-&fspi_pins {
-	bootph-all;
-};
 
 &pinctrl {
 	bootph-all;
@@ -68,34 +64,29 @@
 &sdmmc0 {
 	status = "okay";
 };
+
+&sdmmc1_bus4 {
+    bootph-all;
+};
+&sdmmc1_cmd {
+    bootph-all;
+};
+&sdmmc1_clk {
+    bootph-all;
+};
+
+&sdmmc1 {
+    status = "okay";
+};
 
 &sdhci {
 	cap-mmc-highspeed;
 	mmc-ddr-1_8v;
 	mmc-hs200-1_8v;
-	mmc-hs400-1_8v;
-	mmc-hs400-enhanced-strobe;
-};
-
-&sfc {
-	bootph-all;
-	u-boot,spl-sfc-no-dma;
-	#address-cells = <1>;
-	#size-cells = <0>;
-	status = "okay";
-
-	spiflash: flash@0 {
-		bootph-all;
-		compatible = "jedec,spi-nor";
-		reg = <0>;
-		spi-max-frequency = <24000000>;
-		spi-rx-bus-width = <1>;
-		spi-tx-bus-width = <1>;
-	};
-};
-
-&sdmmc2 {
-	status = "disabled";
+    mmc-hs400-1_8v;
+    mmc-hs400-enhanced-strobe;
+	pinctrl-names = "default";
+	pinctrl-0 = <&emmc_bus8 &emmc_clk &emmc_cmd &emmc_datastrobe>;
 };
 
 &uart1 {
